### PR TITLE
mkfs.f2fs: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/mkfs.f2fs.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# mkfs.f2fs
+
+> Creates a F2FS filesystem inside a partition.
+> More information: <https://manned.org/mkfs.f2fs>.
+
+- Create an F2FS filesystem inside partition 1 on device b (`sdb1`):
```
Comment (fazlearefin) [inline pages/linux/mkfs.f2fs.md:6]:
```suggestion
- Create an F2FS filesystem on a block device (`/dev/sdb1`):
```

I think better to keep it generic and call it *block device*. NVMe drives have very different naming like `nvme0`. The partition can also be made be a logical volume like `/dev/vg0/lv0`.

Comment (Managor) [reply pages/linux/mkfs.f2fs.md:6]: I copied the wording from mkfs.ext4: https://github.com/tldr-pages/tldr/blob/main/pages/linux/mkfs.ext4.md
Now that I look at them. Almost all of the mkfs. commands have the same wording.

Comment (fazlearefin) [reply pages/linux/mkfs.f2fs.md:6]: In that case, I guess okay to leave it as it is. Maybe later we can update all the `mkfs.*` pages and refer to partitions as block devices.

**Review by fazlearefin — APPROVED** [review]

File: pages/linux/mkfs.f2fs.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+`sudo mkfs.f2fs {{/dev/sdb1}}`
+
+- Create an F2FS filesystem with a volume-label:
```
Comment (acuteenvy) [inline pages/linux/mkfs.f2fs.md:10]:
```suggestion
- Create an F2FS filesystem with a volume label:
```

**Review by gutjuri — APPROVED** [review]

File: pages/linux/mkfs.f2fs.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# mkfs.f2fs
+
+> Creates a F2FS filesystem inside a partition.
```
Comment (acuteenvy) [inline pages/linux/mkfs.f2fs.md:3]:
```suggestion
> Creates an F2FS filesystem inside a partition.
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**